### ISSUE TITLE
Make the left nav a numbered list

### ISSUE THREAD

**Issue by awfrancisco** [18F/guides-template#60]
Since this is a guide that must be followed in order and with every step, I think the left nav should be an ordered list and then we can remove the "Now go to the next step" from the bottom of each section. Thoughts?

**Comment by mbland** [comment]
That might make sense, but that means we'll also have to put the numbering in the title of each page, since that's with links the page title to the sidebar highlighting/expansion.

**Comment by awfrancisco** [comment]
The Agile guide has nav numbers but they don't appear on each page. Can we use that same method? https://pages.18f.gov/agile/1-goal.html

**Comment by mbland** [comment]
No. The title and the nav must match, unless we want to make the nav logic even hairier.

Basically, I guess I'm tilting towards "no". I'm not sure the numbers would add much, given that the sidebar already implies a strong ordering, and the upside in experience doesn't appear to me to outweigh the downside in implementation complexity. That, and I think the "Now go to..." bits add a nice conversational touch, though that's admittedly completely an artifact of my poetic license.

That said, I'm a backend dev, not a user researcher and design expert, so I would be interested to hear more thoughts on the subject.

**Comment by mbland** [comment]
FWIW, I'm actually revamping how the nav bar works by adding support for an optional `navlink:` property in 18F/guides-style#32. That way the page title could stay the same, but the `navlink:` could have a numbered version of the same, or a number with a shorter title, etc.

**Comment by awfrancisco** [comment]
:+1:
